exo-deploy/terraform/aws/alb/route53.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_route53_record" "external" {
```

Comment:
We are assigning the public sg to public services and private sg to private ones. Worker services don't need a load balancer sg since they have no load balancer. I had moved those modules to `alb/security_groups/` but we can move it into `ecs-cluster/security-groups.tf` if that's what we agree on.